src/AddTodoForm.js

```diff
@@ -10,4 +10,6 @@ function AddTodoForm() {
     )
```

Comment:
🎨 When you have a JSX element that doesn't have content between it, such as your `<input>` element, you can skip the closing tag, like so:

```js
<input type="text" id="todoTitle"/>
```